addon/styles/addon.less

```diff
@@ -15,7 +15,7 @@
 // ----------------------------------------------------------------------
 
 
-svg {
+svg .chart-viewport {
   // Note: Important for Firefox
   display: block;
```

Comment:
I expect this means `display: block` is applied to the `g` and not the `svg`. Is that ok?